app/src/main/res/layout/activity_leave_form.xml

```diff
@@ -7,4 +7,297 @@
     android:layout_height="match_parent"
     tools:context=".ui.user.LeaveFormActivity">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <ScrollView
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:background="@color/app_background"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="16dp">
+
+            <TextView
+                android:id="@+id/textViewTitle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:fontFamily="@font/epilogue_bold"
+                android:textSize="23sp"
+                android:textFontWeight="600"
+                android:text="@string/apply_for_leave"
+                android:textAppearance="?android:attr/textAppearanceLarge"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
```

Comment:
Give proper ID to all views according to the convention, you can take reference from [here](https://stackoverflow.com/questions/12870537/android-naming-convention)